core-bundle/contao/templates/backend/be_tinyMCE.html5

```diff
@@ -16,26 +16,29 @@ window.tinymce && tinymce.init({
   document_base_url: '<?= Environment::get('base') ?>',
   entities: '160,nbsp,60,lt,62,gt,173,shy',
   branding: false,
+  promotion: false,
   skin: 'tinymce-5',
-  setup: function(editor) {
+  setup: editor => {
     editor.getElement().removeAttribute('required');
+    document.querySelectorAll('[accesskey]').forEach(el => {
+      editor.addShortcut('access+' + el.accessKey, el.id, () => el.click());
+    });
   },
-  init_instance_callback: function(editor) {
+  init_instance_callback: editor => {
+    console.log(document.activeElement.id);
     if (document.activeElement && document.activeElement.id && document.activeElement.id == editor.id) {
       editor.editorManager.get(editor.id).focus();
     }
-    editor.on('focus', function() { Backend.getScrollOffset(); });
+    editor.on('focus', () => Backend.getScrollOffset());
   },
 
   <?php $this->block('picker'); ?>
-    file_picker_callback: function (callback, value, meta) {
+    file_picker_callback: (callback, value, meta) => {
       Backend.openModalSelector({
         'id': 'tl_listing',
         'title': document.getElement('.tox-dialog__title').get('text'),
         'url': Contao.routes.backend_picker + '?context=' + (meta.filetype == 'file' ? 'link' : 'file') + '&amp;extras[fieldType]=radio&amp;extras[filesOnly]=true&amp;extras[source]=<?= $this->source ?>&amp;value=' + value + '&amp;popup=1',
-        'callback': function(table, val) {
-          callback(val.join(','));
-        }
+        'callback': (table, val) => callback(val.join(','))
```

Comment:
using array functions will drop support for IE completely, is that intentional?

Reply:
No, not really. Reverted in 9819d6cfbba5771b096a646f6199792d92a7d0a8.